recipes/fisher/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "fisher" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7519d2c884be6f5844bfde900fd294e98a33fe9a943c6b03800513cd281859c3
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
```

Comment:
This package has compiled code, so you're going to need a `build` section like in:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L36-L39

Also since you're building a c-extension against numpy, please see:

https://conda-forge.org/docs/meta.html#building-against-numpy